lib/index.js

```diff
@@ -19,6 +19,20 @@ var render = require('posthtml-render')
  */
 function PostHTML (plugins) {
   this.plugins = plugins || []
+
+  /**
+   * @example
```

Comment:
There is no need to refer to the name of the process if there is a plugin `posthtml`. 
This is one example of how you can use